docs/architecture/autofill/shadow-dom.md

```diff
@@ -0,0 +1,50 @@
+# Shadow DOM
+
+## Introduction
+
+The Shadow DOM API allows a separate, encapsulated DOM tree to be embedded in a page.
+[From MDN](https://developer.mozilla.org/en-US/docs/Web/Web_Components/Using_shadow_DOM):
```

Comment:
Need to add a line break, so `From MDN` starts in front of the quote.

Reply:
Fixed